src/lexer/tests/lexer_tests.rs

```diff
@@ -69,12 +69,14 @@ fn undefined_pragmas_are_ignored_by_the_lexer() {
 #[test]
 fn registered_pragmas_parsed() {
     let mut lexer = lex(r"
-        {external}{ref}{not_registerd}
+        {external}{ref}{sized}{not_registerd}
```

Comment:
since that 'not_registered' is there, I think the test should do something with it.

its a comment right? so maybe check that there are not tokens left?